tests/Unit/Filters/Audio/AudioImageVideoFilterTest.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace Tests\FFMpeg\Unit\Filters\Audio;
+
+
+use FFMpeg\Filters\Audio\AudioFilters;
+use Tests\FFMpeg\Unit\TestCase;
+
+class AudioImageVideoFilterTest extends TestCase
+{
+    public function testAddImageVideo()
+    {
+        $artwork = '/path/to/artwork.jpg';
+
+        $audio = $this->getAudioMock();
+        $audio->expects($this->once())
+            ->method('addFilter')
+            ->with($this->isInstanceOf(\FFMpeg\Filters\Audio\ImageVideoFilter::class))
+            ->will($this->returnCallback(function ($filter) use (&$capturedFilter) {
+                $capturedFilter = $filter;
+            }));
+
+        $format = $this->getMockBuilder(\FFMpeg\Format\AudioInterface::class)->getMock();
+        $filter = new AudioFilters($audio);
+
+        $filter->imageVideo($artwork);
+        $this->assertEquals([ '-loop', 1, '-i', '/path/to/artwork.jpg', '-preset', 'veryslow', '-shortest' ], $capturedFilter->apply($audio, $format));
```

Comment:
please replace `1` with `'1'`, as it will be replaced later and is more clean for ffmpeg, while I see that a numeric value is always awesome here.